lib/backend.js

```diff
@@ -83,7 +82,7 @@ SequelizeBackend.prototype = {
 			return this.findRow(bucket);
 		}
 		return this.getModel(bucket).findAll({
-			where: {key: {$in: keys.map(function(key){ return key.toString();})}},
+			where: {key: {[this.db.Op.in]: keys.map(function(key){ return key.toString();})}},
```

Comment:
I think we can save a reference to Op in the constructor and then access it buy `this.Op` might be easier to refactor in the future.